project/build.scala

```diff
@@ -151,7 +151,14 @@ object build {
     }),
     scalacOptions ++= {
       CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, 11 | 12)) =>
+        case Some((2, 12)) =>
+          Seq(
+            "-Xsource:2.12",
+            "-opt:l:method,inline",
+            "-opt-inline-from:scalaz.**"
```

Comment:
@edmundnoble brings up: do we want to inline between modules? The argument against being that something spooky might happen if I depend on `scalaz-core 7.2.20` and `scalaz-effect 7.2.14` or similar.

Reply:
fix your dependencies @hrhino 

![](https://media.giphy.com/media/Qvwc79OfQOa4g/giphy.gif)

Reply:
![...](http://78.media.tumblr.com/834412ae7eb8db47a32fdd7b3bf0daf7/tumblr_ns2tfoWgrl1syy397o3_400.gif)

Reply:
wait hold up, I didn't say they're _mine_! @edmundnoble's the one who [brought it up](https://gitter.im/scalaz/scalaz?at=5ac402b67c3a01610da5209d)!